lib/ansible/plugins/filter/password_hash.yml

```diff
@@ -16,9 +17,9 @@ DOCUMENTATION:
       description: Hashing algorithm to use.
       type: string
       default: sha512
-      choices: [ md5, blowfish, sha256, sha512 ]
+      choices: [ md5, bcyrpt, blowfish, sha256, sha512 ]
```

Comment:
If `bcrypt` is added here, it should also be added as `'bcrypt': 'bcrypt',` in https://github.com/ansible/ansible/blob/d00d065aaf98d608ed9a6c27b7beaa8ad278c2a1/lib/ansible/plugins/filter/core.py#L278-L283. Otherwise the error message in https://github.com/ansible/ansible/blob/d00d065aaf98d608ed9a6c27b7beaa8ad278c2a1/lib/ansible/plugins/filter/core.py#L305 will not list `bcrypt`.

Reply:
From the comments in the User Help channel, it seems that using `bcrypt` instead of `blowfish` does work. However, now that I see how the code is structured, I agree it should not be listed as a value (unless the other right-hand values would also work and we wanted to list them all, which I do not).

Thanks for catching that @felixfontein!

Reply:
The values on the right-hand side also work, see the `hashtype not in passlib_mapping.values()` part of https://github.com/ansible/ansible/blob/d00d065aaf98d608ed9a6c27b7beaa8ad278c2a1/lib/ansible/plugins/filter/core.py#L288, but they are not listed as acceptable values in the error message in line 305.

Personally I would avoid adding the `xxx_crypt` values, but adding `bcrypt` sounds like a good idea to me since that name is well-known.